packages/ckeditor5-enter/src/enter.js

```diff
@@ -38,7 +38,11 @@ export default class Enter extends Plugin {
 		editor.commands.add( 'enter', new EnterCommand( editor ) );
 
 		this.listenTo( viewDocument, 'enter', ( evt, data ) => {
-			data.preventDefault();
+			// Rendering is disabled while composing so prevent events that will be rendered by the engine
```

Comment:
I'd reword it because I had troubles understanding this comment. Maybe something like:

> When not in composition, we handle the action, so prevent the default one.
> When in composition, it's the browser who modify the DOM (renderer is disabled).